modules/reference/pages/rpk/rpk-cloud/rpk-cloud-cluster-select.adoc

```diff
@@ -0,0 +1,42 @@
+= rpk cloud cluster select
+
+Update your rpk profile to talk to the requested cluster.
+
+This command is essentially an alias for the following command:
+
+```bash
+rpk profile create --from-cloud=${NAME}
+```
+
+If you want to name this profile rather than creating or updating values in the default cloud-dedicated profile, you can use the `--profile` flag.
+
+== Usage
+
+[,bash]
+----
+rpk cloud cluster select [NAME] [flags]
+----
+
+== Aliases
+
+[,bash]
+----
+select, use
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|-h, --help |- |Help for select.
+
+|--profile |string |Name of a profile to create or update (avoids updating "rpk-cloud").
```

Comment:
@r-vasquez

Reply:
`rpk cloud cluster select` selects a cloud cluster, and saves the profile to the `rpk-cloud` profile (i.e the default cloud-dedicated profile). By using `--profile`, you name the profile with the selected cluster instead of updating `rpk-cloud`.

The help text contains this paragraph:
```
If you want to name this profile rather than creating or updating values in
the default cloud-dedicated profile, you can use the --profile flag.
```